sdks/java/core/src/main/java/org/apache/beam/sdk/schemas/AvroRecordSchema.java

```diff
@@ -32,12 +32,14 @@
  * objects, creating a schema that matches that inferred by the AVRO libraries.
  */
 @SuppressWarnings({
+  "nullness", // TODO(https://issues.apache.org/jira/browse/BEAM-10402)
```

Comment:
Why? It would be best instead to not introduce nullness errors to this file.

Reply:
`avroType.getClassLoader()` can return null apparently. It is done that way everywhere else but here.

Reply:
Does `ReflectData` require its argument to be non-null?

Reply:
No, the default constructor actually calls with `null`. 
There is however no `@Nullable` annotation in the library. Seems the checker considers it as required.

Reply:
Makes sense. At some point we should improve upstream apache/avro or possibly just add a stub file here. For now, can you narrow the scope of the `@SuppressWarnings` annotation to this method? Or, ideally, to a single assignment statement like so:

```java
Class<?> avroType = typeDescriptor.getRawType();
@SuppressWarnings("nullness") // new ReflectData(ClassLoader) is not annotated to accept null
ClassLoader classLoader = avroType.getClassLoader
return toBeamSchema(new ReflectData(classLoader).getSchema(avroType));
```